src/wp-admin/includes/template.php

```diff
@@ -1700,6 +1733,10 @@ function do_settings_sections( $page ) {
 	}
 
 	foreach ( (array) $wp_settings_sections[ $page ] as $section ) {
+		if ( '' !== $section['before_section'] ) {
+			echo sprintf( $section['before_section'], $section['section_class'] );
+		}
```

Comment:
A little extra check could help in the event there's no class attribute to render:

```php
if ( '' !== $section['before_section'] ) {
	if ( '' !== $section['section_class'] ) {
		printf( $section['before_section'], esc_attr( $section['section_class'] ) );
	} else {
		echo $section['before_section'];
	}
}
```

Shouldn't `'before_section'` be escaped before echoing out to the browser?

Reply:
I added the neat little check to handle the missing `section_class`.